src/components/StatusPageNotMultiple.js

```diff
@@ -14,7 +14,7 @@ import { ReasonAndTypeForm } from '@/components/ReasonAndTypeForm';
 
 import { OfferStatus } from '@/constants/OfferStatus';
 
-const StatusFormOnPage = ({ offer, error, useChangeStatus }) => {
+const StatusPageNotMultiple = ({ offer, error, useChangeStatus }) => {
```

Comment:
So... single? 😛 

I think I get the reasoning, because it's also the page for `periodic` and `permanent`. But in this context we can maybe just use `single` since it's a "single status"?

Reply:
Suggestion: `SingleStatusPage` and `MultipleStatusPage`?